lips/lip-6.md

```diff
@@ -0,0 +1,79 @@
+
+# Terminology
+* *Validator Node* ("node") is a replica that runs the Libra Protocol to maintain a database of programmable resources. Validator Node processes transactions and interacts with other Validator Nodes to reach consensus on the ordering of transactions and the resulting state of the database after transactions' executions. 
+* *Validator Owner* is a entity that manages an on-chain Libra account of the member of the Libra Association. The account stores the *Validator Config* resource that contains information about the Validator Node of this Validator Owner. This information includes *consensus public key* (EdDSA public key that is used by others to authenticate consensus messages of this node) and *networking information* (network addresses and static Noise keys for establishing secure connections with this node from other nodes or from clients).
+* *Validator Set* is a copy of Validator Configs of the approved subset of the Validator Owners, it consists of configs currently running the Libra Protocol. The Libra Association collectively decides which Validator Owners' configs are in the set. There is a one-to-one correspondence between the Validator Owners in the Validator Set and the Validator Nodes. The protocol remains safe when at most a third minus one of the Validator Nodes are byzantine.
+* *Validator Operator* ("operator") is an entity that manages its on-chain Libra account and runs a Validator Node on behalf of the Validator Owner who designated this operator to be running the node. There can be one Validator Operator account on-chain that runs two Validator Nodes for two Validator Owners, one node per one owner. It is not allowed to run a single Validator Node per multiple Validator Owners even when the nodes are run by the same Validator Operator. The Validator Operator has the permission to rotate the Validator Configs for its Validator Owners.
+
+# Specification
+## Overview
+Each Validator Owner holds a Libra account with a *Validator Owner role* (see [lip-2: Libra Roles and Permissions](https://github.com/libra/lip/blob/master/lips/lip-2.md)). A Validator Owner designates a Validator Operator to run a Validator Node
```

Comment:
I think we can have consistent and precise language in all of these specifications.  I'm not sure we're all using the same language (we should agree as a group).
Here is how I've been thinking of these concepts in formal specs
1. The Move framework represents Validator Nodes and Validator Operators as addresses.  I'm not sure what terminology others use, but an address that has a published account resource could be called an "account".
2. Every account has a unique role (see roles and permissions)
3. Each Validator account has a Validator role. 
4. Each ValidatorOperator account has a ValidatorOperator role.

Reply:
Great, I was thinking of using this more formal language in the subsection further down, called ## Formal verification guarantees on the Move Implementation. I can add those points there. But I am not quite sure about the terminology, may be we can discuss it over the call.